type.go

```diff
@@ -57,18 +70,23 @@ func (ht *HsTime) UnmarshalJSON(b []byte) error {
 }
 
 // String implemented Stringer.
-// If the value is zero, it will be displayed as `<nil>`.
 func (ht *HsTime) String() string {
+	if ht == nil {
+		return "nil"
```

Comment:
If this returns `"nil"`, what happens  on hubspot.NewString("nil")  when the value is extracted from HubSpot?

```go
v := time.Time(*ht)
v.IsZero()  // => ture?
```
Can you add the test case?

Reply:
actually it's gonna be ` NewTime(t time.Time)` so `"nil"` won't be consumed in HsTime as long as it is not registered in HubSpot...!